### DISCUSSION TITLE
Why does Content Deployment export only Published items?

### DISCUSSION THREAD

**Discussion by SzymonSel** [OrchardCMS/OrchardCore#14246] [General]
Is this by design?

**Comment by Piedone** [comment]
Yes. The logic in all places, including the code and a lot of UI, is that by default, you only see/fetch published content items. We could add an option to export all the latest content items though, regardless of their version flag.

**Comment by SzymonSel** [comment]
Right, thanks for the explanation! I think this should be an options, thus it might not always be obvious. I'll have a swing at it, since I'm already somewhat "accustomed" with deployments ;)

**Reply by Piedone** [reply]
Sounds good, would be nice! Perhaps the same for All Content too.

**Reply by SzymonSel** [reply]
Sure!
